pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/PulsarAdmin.java

```diff
@@ -187,22 +187,23 @@ public PulsarAdmin(String serviceUrl,
                 Math.toIntExact(readTimeoutUnit.toMillis(this.readTimeout)),
                 Math.toIntExact(requestTimeoutUnit.toMillis(this.requestTimeout))).getHttpClient();
 
-        this.clusters = new ClustersImpl(root, auth);
-        this.brokers = new BrokersImpl(root, auth);
-        this.brokerStats = new BrokerStatsImpl(root, auth);
-        this.tenants = new TenantsImpl(root, auth);
-        this.properties = new TenantsImpl(root, auth);;
-        this.namespaces = new NamespacesImpl(root, auth);
-        this.topics = new TopicsImpl(root, auth);
-        this.nonPersistentTopics = new NonPersistentTopicsImpl(root, auth);
-        this.resourceQuotas = new ResourceQuotasImpl(root, auth);
-        this.lookups = new LookupImpl(root, auth, useTls);
-        this.functions = new FunctionsImpl(root, auth, httpAsyncClient);
-        this.sources = new SourcesImpl(root, auth, httpAsyncClient);
-        this.sinks = new SinksImpl(root, auth, httpAsyncClient);
-        this.worker = new WorkerImpl(root, auth);
-        this.schemas = new SchemasImpl(root, auth);
-        this.bookies = new BookiesImpl(root, auth);
+        long readTimeoutMs = readTimeoutUnit.toMillis(this.readTimeout);
```

Comment:
I haven't found the line which sets `readTimeout` in https://github.com/apache/pulsar/blob/master/pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java , so `readTimeout` seems to be set to the default value (60 seconds) for Brokers.
https://github.com/apache/pulsar/blob/b5d779db5f674469797873f2c5627d4805aae082/pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/PulsarAdmin.java#L72

Can you make it configurable in `broker.conf`?
(I feel 60 seconds is too large for Brokers)

Reply:
sure, I will fix it.